gp-templates/translations.php

```diff
@@ -34,7 +34,7 @@
 		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
 		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
 	<?php if( $can_write ) : ?>
-		<option value="set-priority"><?php _e( 'Set Priority', 'glotpress' ); ?></option>
+		<option value="set-priority" class="hide-if-no-js"><?php _e( 'Set Priority', 'glotpress' ); ?></option>
```

Comment:
Should we hide this without JS?  A non-JS user would never be able to set the bulk priority if we do.  Though, I think it's safe to say that JS is a requirement for GP :)